widgetssdk/src/main/res/values/themes_material.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why this addition?

Reply:
Paparazzi wants a Material theme for rendering the Material views.
```
java.lang.IllegalArgumentException: The style on this component requires your app theme to be Theme.MaterialComponents
```

Reply:
This file should be moved to a separate directory with flavor or something else. It will be implemented in the next Screenshot testing task.